generators/entity-server/files.js

```diff
@@ -192,7 +198,7 @@ const serverFiles = {
             ]
         },
         {
-            condition: generator => generator.reactive && ['mongodb', 'cassandra', 'couchbase'].includes(generator.databaseType),
+            condition: generator => generator.reactive && !generator.embedded,
```

Comment:
Removing `['mongodb', 'cassandra', 'couchbase'].includes(generator.databaseType)` was on purpose?

Reply:
No, I didn't pay attention, it was a mistake after a review commit...